Bottom: cd8cab0ac07255aa19fc973f66f4d8506781392d
Top:    5bf982aeff368b997ae5d1fb6ec34975051b0176
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-10-03 17:36:36 -0700

fixup


---
diff --git a/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py b/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
index 42572a32bd4..4db9c7dcf48 100644
--- a/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
+++ b/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
@@ -754,7 +754,7 @@ class FileHandleTestCase(lldbtest.TestBase):
         with open(self.out_filename, 'w') as f:
             status = debugger.SetOutputFile(f)
             self.assertTrue(status.Success())
-            self.handleCmd(debugger, "help help", collect_result=False)
+            self.handleCmd("help help", check=False, collect_result=False)
             # make sure the file wasn't closed early.
             f.write("\nZAP\n")
             lldb.SBDebugger.Destroy(debugger)
